### ISSUE TITLE
AADGroupsSettings: Resources using Select-MgProfile -Name 'beta' fail on Test/Start-DSCConfiguration 

### ISSUE THREAD

**Issue by afloca** [microsoft/Microsoft365DSC#1586]
#### Details of the scenario you tried and the problem that is occurring
Hello,

At some point Resources using the Beta Graph API stoped working (perhaps as more functions became available in the Graph beta SDK)
Export of such resources works fine, however when attempting to test or apply the configuration the $MaximumFunctionCount Limit of Powershell 5.1 is reached
Using PS Core where this limit was removed would not make a difference pecause LCM then would still use PS 5.1. On the other hand not using the Beta profile will also not help as the commandlets are still not avialable under 1.0

Could not find a way to set the $MaximumFunctionCount in the session where LCM is involved/maybe am missing something here?
The other part would be the question how this can be mitigated when now working resources that were migrated to the Graph SDK, once the number of functions in the 1.0 Graph surpasses the limitation?

Best,
Alex
#### Verbose logs showing the problem
CloseError: Function Remove-MgUserOutlookTaskGroupTaskFolderTaskAttachment cannot be created because function capacity 4096 has been exceeded for this scope.
InvalidOperation: The PowerShell DSC resource '[AADGroupsSettings]' with SourceInfo 'C:\temp\group\M365TenantConfig.ps1::15::9::AADGroupsSettings' threw one or more non-terminating errors while running the Test-TargetResource functionality. These errors are logged to the ETW channel called Microsoft-Windows-DSC/Operational. Refer to this channel for more details.

MIResult: 1
Error Message: Function Remove-MgUserOutlookTaskGroupTaskFolderTaskAttachment cannot be created because function capacity 4096 has been exceeded for this scope.
Message ID: Microsoft.Graph.PowerShell.Authentication.Cmdlets.SelectMgProfile
Error Category: 2
Error Code: 2
Error Type: MI

#### The DSC configuration that is used to reproduce the issue (as detailed as possible)
```powershell
        AADGroupsSettings e92cc028-22d9-4494-a880-f7778a85065f
        {
            AllowGuestsToAccessGroups     = $True;
            AllowGuestsToBeGroupOwner     = $False;
            AllowToAddGuests              = $True;
            ApplicationId                 = $ConfigurationData.NonNodeData.ApplicationId;
            CertificateThumbprint         = $ConfigurationData.NonNodeData.CertificateThumbprint;
            EnableGroupCreation           = $False;
            Ensure                        = "Present";
            GroupCreationAllowedGroupName = "AS-AllowCreation-User";
            GuestUsageGuidelinesUrl       = "";
            IsSingleInstance              = "Yes";
            TenantId                      = $ConfigurationData.NonNodeData.TenantId;
            UsageGuidelinesUrl            = "";
	    PsDscRunAsCredential          = $PSDSCcred
        }
```

#### Version of the DSC module that was used ('dev' if using current dev branch)
1.21.1117.2

**Comment by NikCharlebois** [comment]
@afloca we did some work around this in version 1.21.1124.1. Can you please try upgrading to that version and confirm whether or not you are still getting the error. Thanks

**Comment by afloca** [comment]
@NikCharlebois have upgraded to 1.21.1124.1 however the beaviour is the same:
![image](https://user-images.githubusercontent.com/80976204/143577161-d1ee2ea0-5414-45f2-b438-10a4ee40a153.png)

**Comment by NikCharlebois** [comment]
This is fixed in 1.21.1208.1 can you please confirm that it is now working for you? Thanks

**Comment by afloca** [comment]
I can confirm this is working now. Thanks! Alex.
